css.md

```diff
@@ -0,0 +1,32 @@
+# CSS
+
+**Prerequisite knowledge**: basic html elements
+
+## Intro
+CSS (Cascading Style Sheets) is a style sheet language used for describing the presentation of document written in a markup language (like HTML).
+	CSS can improve content accessibility and readability by affecting layout, colors and fonts. Also  provide more flexibility and control in the specification of presentation characteristics.
+	Enable multiple web pages to share formatting by specifying the relevant CSS in a separate .css file, and reduce complexity and repetition in the structural content. 
+	The name cascading comes from the specified priority scheme to determine which style rule applies if more than one rule matches a particular element. 
+	This cascading priority scheme is predictable. [source](https://en.wikipedia.org/wiki/Cascading_Style_Sheets)
+
+## Classification / acceptance questions
+
+### Apprentice aka _Short skillset description, example: "I can branch!", "My First TODO app"_
+	*I know what is a box model, and I know its basic elements (graphical interpretation)
+	*I know how html elements could be displayed (in line, in line block, flex, table)
+	*I know what is the difference between in line, in line block and flex
+	*I know whats are classes
+	*I know how inheritance works
+	*I can change colors and fonts of text
+	*I can use padding, margin, and border attributes
+	*I can modify elements by ID
```

Comment:
Modify or apply styles? If the latter, then I'd merge it with the one about classes.

Reply:
I would create a more general point about writing css selectors and there you can define different selectors:
- by tag name
- by class
- by id
and more complex like:
- by siblings
- by attributes

I would mention somewhere the specifity of selectors (https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)